### PR TITLE
Add support to newer python version

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description

Add support to newer python version

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/cli.py [hunk 1]
```diff
@@ -46,10 +46,8 @@
 
 
 def check_python_version():
-    if sys.version_info >= (3, 11):
-        raise RuntimeError("Python versions 3.11 and above are not yet supported. Please use Python 3.8, 3.9 or 3.10.")
-    if sys.version_info < (3, 8):
-        raise RuntimeError("Python versions 3.7 and below are not supported. Please use Python 3.8, 3.9 or 3.10")
+    if sys.version_info < (3, 9):
+        raise RuntimeError("Python versions 3.8 and below are not supported. Please use Python >= 3.9")
```
[the remark this answers is not in the document]
Comment (YuanTingHsieh) [reply nvflare/cli.py]: thanks, fixed and porting your PR over

File: .github/DISCUSSION_TEMPLATE/q-a.yml [hunk 2]
```diff
@@ -9,26 +9,26 @@ body:
     attributes:
       label: Python version (`python3 -V`)
       description: Python version (`python3 -V`)
-      placeholder: "3.8"
+      placeholder: "3.10"
```
Comment (yhwen) [inline .github/DISCUSSION_TEMPLATE/q-a.yml:12]: I think we should not have added these contents in the ".github" folder into the repo. These contents are generated by github after the project set up, no need to maintain in the repo ourselves.

Comment (yhwen) [reply .github/DISCUSSION_TEMPLATE/q-a.yml:12]: Discussed with @YuanTingHsieh, we need to have these templates to enable the customized discussions and issues.

**Review by yhwen — APPROVED** [review]

**Pull request merged**